src/Microsoft.Azure.WebJobs.Extensions.Kafka/Output/IKafkaProducer.cs

```diff
@@ -13,5 +13,15 @@ public interface IKafkaProducer : IDisposable
         /// Produces a Kafka message
         /// </summary>
         Task ProduceAsync(string topic, object item);
+
+        /// <summary>
+        /// Produces a Kafka message
+        /// </summary>
+        void Produce(string topic, object item);
```

Comment:
Why did you switch the sychronous method?

Reply:
We checked the documentation and they had written that Produce method is more tightly bound to Produce method of librdk Kafka and thus they recommend to use Produce method in sync mode for better performance.
We also did the metric and our result is quiet very positive with this approach thus we moved on this API.
You can check the document that I shared offline for metrics